src/main/scala/org/broadinstitute/dsde/agora/server/model/AgoraEntity.scala

```diff
@@ -106,6 +106,13 @@ object AgoraEntity {
     (namespace |@| name |@| _id |@| synopsis |@| doc) {(namespace, name, _id, synopsis, doc) => doNothing }
   }
 
+  def getErrors(entity: AgoraEntity): Option[IllegalArgumentException] = {
```

Comment:
it feels weird that only tests are using this because then we're testing code that isn't actually ever used in production